package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "restaurant_page",
+  "version": "1.0.0",
+  "description": "In this project we continue practicing DOM manipulation by dynamically rendering a simple restaurant homepage. By the end, we are going to be using JavaScript alone to generate the entire contents of the website!",
+  "main": "index.js",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "stylelint": "^13.3.3",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "stylelint-config-standard": "^20.0.0",
+    "css-loader": "^5.1.2",
+    "file-loader": "^6.2.0",
+    "style-loader": "^2.0.0",
+    "webpack": "^5.24.4",
+    "webpack-cli": "^4.5.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
Good work adding web-pack to your project 👍🏿 👏🏿  , However, there are some configuration missing ⬇️ 

- [ ] Kindly configure well the necessary run common commands like `build`, `development`.  This will ensure that different commands can be issued to your application. You can configure them like the example below ⬇️ 
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
![scripts_requirement](https://user-images.githubusercontent.com/32361963/110933272-9a4e5280-833d-11eb-8c13-97cf9bf24c43.png)